mne/channels/tests/test_layout.py

```diff
@@ -7,12 +7,12 @@
 
 import copy
 import os.path as op
-import matplotlib
 
 import numpy as np
 from numpy.testing import (assert_array_almost_equal, assert_array_equal,
                            assert_allclose, assert_equal)
 import pytest
+import matplotlib.pyplot as plt
```

Comment:
why do you need this?

Reply:
It was moved up from below in test_find_layout

Reply:
(Eventually there is lout.plot and plt.close)